__tests__/core/gatherer.test.ts

```diff
@@ -42,12 +43,61 @@ describe('Gatherer', () => {
     await server.close();
   });
 
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
   it('boot and close browser', async () => {
     const driver = await Gatherer.setupDriver({ wsEndpoint });
     expect(typeof driver.page.goto).toBe('function');
     await Gatherer.stop();
   });
 
+  // This test should only run when a browser service is up
+  (wsEndpoint ? it : it.skip)(
+    'does not the disable-gpu flag to start browser when running headful',
+    async () => {
+      if (!wsEndpoint) return;
```

Comment:
Having this should be enough to skip the tests right? Do we need `it or it.skip`?

Reply:
True, it should be enough, it can be removed.